wdl2cwl/tests/oop_cwl_files/CollectQualityYieldMetrics.cwl

```diff
@@ -0,0 +1,38 @@
+class: CommandLineTool
+id: CollectQualityYieldMetrics
+inputs:
+  - id: input_bam
+    type: File
+  - id: metrics_filename
+    type: string
+  - id: preemptible_tries
+    type: int
+outputs:
+  - id: quality_yield_metrics
+    type: File
+    outputBinding:
+        glob: $(inputs.metrics_filename)
+requirements:
+  - class: DockerRequirement
+    dockerPull: us.gcr.io/broad-gotc-prod/picard-cloud:2.23.8
+  - class: InitialWorkDirRequirement
+    listing:
+      - entryname: example.sh
+        entry: |4
+
+            java -Xms2000m -jar /usr/picard/picard.jar \
+              CollectQualityYieldMetrics \
+              INPUT=$(inputs.input_bam.path) \
+            OQ=true \
+            OUTPUT=$(inputs.metrics_filename)
+            sed -i -e 1,5d $(inputs.metrics_filename)  # for reproducibility
+  - class: InlineJavascriptRequirement
+  - class: NetworkAccess
+    networkAccess: true
+  - class: ResourceRequirement
+    ramMin: 3584.0
+    outdirMin: 40
```

Comment:
```suggestion
    outdirMin: 40960
```
for WDL, `runtime.disks` defaults to `GiB` units if not specified

https://github.com/openwdl/wdl/blob/main/versions/1.1/SPEC.md#disks